tests/fixtures/ca-subdiff.json

```diff
@@ -0,0 +1,330 @@
+{
+    "element": "shoji:view",
+    "self": "https://app.crunch.io/api/datasets/6c245c/cube/?query=%7B%22dimensions%22%3A%5B%7B%22function%22%3A%22dimension%22%2C%22args%22%3A%5B%7B%22variable%22%3A%22https%3A%2F%2Fapp.crunch.io%2Fapi%2Fdatasets%2F6c245c%2Fvariables%2Ff054e4%2F%22%7D%2C%7B%22value%22%3A%22subvariables%22%7D%5D%7D%2C%7B%22variable%22%3A%22https%3A%2F%2Fapp.crunch.io%2Fapi%2Fdatasets%2F6c245c%2Fvariables%2Ff054e4%2F%22%7D%5D%2C%22measures%22%3A%7B%22count%22%3A%7B%22function%22%3A%22cube_count%22%2C%22args%22%3A%5B%5D%7D%7D%2C%22weight%22%3Anull%7D&filter=%7B%7D",
+    "value": {
+        "query": {
```

Comment:
Yeah, I kind of knew this was coming. For dev purposes I like having the file with real interpretations (and that I have control of), but I can change to use existing fixtures before finalizing.

Reply:
There's no problem with adding this fixture if you feel it is distinctive, just name it `ca-subvar-x-ca-cat.json` since we don't have one of those yet.